application/templates/cms_autosave/macros.html

```diff
@@ -269,20 +231,21 @@
 {%- endmacro %}
 
 
-{% macro editable_department_select(field, value, organisations_by_type) -%}
+{% macro editable_department_select(field, organisations_by_type, value_text=None, value_id=None) -%}
     <div class="preview-and-edit">
         <p class="empty preview" data-empty-text="{{ kwargs['empty_message'] }}" tabindex={{ kwargs['tab_index'] }}>
-        {% if value -%}
-            {{ value }}
+        {% if value_text -%}
+            {{ value_text }}
         {% else %}
             {{ kwargs['empty_message'] }}
         {%- endif %}
         </p>
         <div class="edit hidden">
             {{ render_department_select('',
                                         field,
-                                        measure.department_source_id if measure else None,
-                                        organisations_by_type) }}
+                                        value_id if value_id else None,
```

Comment:
Do we need this condition, seeing as the macro defaults `value_id` to `None` if unspecified anyway?

Reply:
Possibly not.